e2e/vm/config_darwin_test.go

```diff
@@ -33,9 +32,10 @@ var testConfig = func(o *option.Option, installed bool) {
 			}
 
 			limaConfigFilePath := resetVM(o, installed)
+			resetDisks(o, installed)
 			writeFile(finchConfigFilePath, []byte("memory: 4GiB\ncpus: 6\nvmType: vz\nrosetta: false"))
-			initCmdSession := command.New(o, virtualMachineRootCmd, "init").WithTimeoutInSeconds(600).Run()
-			gomega.Expect(initCmdSession).Should(gexec.Exit(0))
+			// vm init with VZ set sometimes takes 2 minutes just to convert the disk to raw
+			command.New(o, "vm", "init").WithoutCheckingExitCode().WithTimeoutInSeconds(240).Run()
```

Comment:
Saw a bit of discussion regarding the exit code stuff, but still don't quite understand, why would we want to ignore exit codes that are non 0?

Reply:
I think there's a small chance of the error code being non-zero, yet completing before the timeout. In this case, if the functionality that's being tested still works (i.e. all the stuff _after_ the `vm init` happens), I think that's good enough.

The `vm` commands has its own tests where the error codes are not ignored. Happy to discuss further.

Reply:
That's pretty strange, so you're saying like a partially successful vm init can still allow the e2e tests to still succeed? (ie maybe everything is good except something like networking or mounting a certain directory fails?)

Reply:
> The vm commands has its own tests where the error codes are not ignored. Happy to discuss further.

Yeah thats fair, guess it's not crucial for it here since that's not what we're testing. I wonder how much vm failures affect non-vm related tests though.